code/modules/mob/living/flavortext.dm

```diff
@@ -0,0 +1,111 @@
+/mob/living
+	var/adjectives = list(
```

Comment:
You're giving EVERY LIVING MOB a DISTINCT copy of this list?

Maybe you should go `static` or something